src/checkout/checkout-selector.ts

```diff
@@ -12,6 +12,7 @@ import CheckoutState, { DEFAULT_STATE } from './checkout-state';
 export default interface CheckoutSelector {
     getCheckout(): Checkout | undefined;
     getGrandTotal(useStoreCredit?: boolean): number | undefined;
```

Comment:
Is it possible to remove this getter and replace it with `getOutstandingBalance`? Also, to make it backwards compatible, modify `getOutstandingBalance` so it accepts `useStoreCredit`. If `useStoreCredit` flag is not passed, return `checkout.outstandingBalance`. Otherwise, return what `getGrandTotal` is currently returning.